src/business-logic/services/test/device-changed-service.spec.ts

```diff
@@ -1,16 +1,16 @@
 import { DeviceChangedService } from '../device-changed-service'
 import { anything, capture, instance, mock, when } from '@typestrong/ts-mockito'
 import { DataChangedListener } from '../../../data-access/repositories/interfaces/data-changed-listener'
-import { Device } from '../../../model/entities/device'
-import { DeviceRepository } from '../../../data-access/repositories/interfaces/device-repository'
 import { Logger } from '../../../logger/logger'
 import { StatusMessageService } from '../interfaces/status-message-service'
-import { EntityTestFactory } from '../../../model/entities/test/entity-test-factory'
 import { StatusCode } from '../../../model/enums/status-code'
+import { Device } from '../../../model/entities/device'
+import { DeviceRepository } from '../../../data-access/repositories/interfaces/device-repository'
+import { EntityTestFactory } from '../../../model/entities/test/entity-test-factory'
 
 describe(DeviceChangedService.name, () => {
   it('calls the StatusMessageService with a StatusMessage that has the same id as the Device', () => {
-    const device: Device = EntityTestFactory.createDevice({ id: 'deviceId' })
+    const device: Device = mock<Device>() //EntityTestFactory.createDevice({ id: 'deviceId' })
```

Comment:
What's the reason for uncommenting `EntityTestFactory.createDevice(...)`?

Reply:
All tests parses when using the current implementation of `TestEntityFactory.createDevice()`.